src/patterns/feedback/examples/feedback-form/index.njk

```diff
@@ -0,0 +1,142 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/textarea/_macro.njk" import onsTextarea %}
+{% from "components/button/_macro.njk" import onsButton %}
+
+{% set pageConfig = {
+    "title": "Census 2021",
+    "theme": "census",
+    "absoluteUrl": pageInfo.url,
+    "headMeta": {
+        "description": "Census 2021 is a UK wide population survey"
+    },
+    "header": {
+        "logoHref": "#0",
+        "titleLogo": "census-logo-en",
+        "titleLogoHref": "#0",
+        "titleLogoAlt": "Census 2021"
+    },
+    "breadcrumbs": {
+        "ariaLabel": 'Back',
+        "itemsList": [
+            {
+                "url": '/',
+                "id": "back",
+                "text": 'Back',
+                "attributes": {
+                    "data-attribute": "Example attribute"
+                }
+            }
+        ]
+    },
+    "footer": {
+    }
+} %}
+{% block main %}
+    <h1>Feedback</h1>
```

Comment:
```suggestion
    <h1>Give feedback about this service</h1>
```

Reply:
I think the feedback component is a valid component though. It has it's own macro.

Reply:
I spoke to Jon about this and I think it makes sense for all the feedback docs to be in the same place as its all part of the same pattern